### PR TITLE
final commit

### PR DESCRIPTION
**Pull request by nehalskdev**

### CODE REVIEW

**Comment by ManSOorcode** [comment]
what about this ?

ezgif com-video-to-gif

**Review by ManSOorcode — APPROVED** [review]
It look good but you can improve like button can take border radius transition will be get smooth take look which I have suggest you

**Comment by nehalskdev** [comment]
> It look good but you can improve like button can take border radius transition will be get smooth take look which I have suggest you

I didn't understand your point i will connect with you on duo in your available time.😁

**Comment by MTahir001** [comment]
Looks good brother
u can make it better using some other font

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          </div>
+          <div class="color">
+            <h3>color:</h3>
+            <span id="color-1"></span>
+            <span id="color-2"></span>
+            <span id="color-3"></span>
+          </div>
+          <a
+            href="https://www.nike.com/in/t/court-vision-low-next-nature-shoes-N2fFHb/DH2987-101"
+            target="#"
```
Comment (murtazshaikh) [inline index.html:33]: Something exists like this target="#"

File: style.css [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+#color-2 {
+  background: #03a9f4;
+}
+#color-3 {
+  background: #e91e63;
+}
+.container .card .contentbx a {
+  display: inline-block;
+  padding: 10px 20px;
+  margin: 10px 0 0 0;
```
Comment (murtazshaikh) [inline style.css:176]: can use only margin-top

File: style.css [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+  transition-delay: 0.5s;
+}
+.container .card:hover .contentbx .color {
+  opacity: 1;
+  visibility: visible;
+  transition-delay: 0.6s;
+}
+.container .card .contentbx .size h3,
+.container .card .contentbx .color h3 {
+  margin: 0 10px 0 0;
```
Comment (murtazshaikh) [inline style.css:124]: same ?

File: style.css [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+  position: absolute;
+  bottom: 0;
+
+  text-align: center;
+  transition: 1s;
+  z-index: 10;
+}
+.container .card:hover .contentbx {
+  height: 210px;
```
Comment (murtazshaikh) [inline style.css:93]: always avoid setting height, when we place things inside container height automatically gets adjusted

**Review by murtazshaikh — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (nehalskdev) [reply index.html:33]: Yes it does exist and I've used it many times. As i don't want the user to launch the link provided in my anchor tag on the same page i want the link to launch on new tab

File: style.css (hunk 2 above)
Comment (nehalskdev) [reply style.css:176]: It's nothing but a shorthand for that.

File: style.css (hunk 3 above)
Comment (nehalskdev) [reply style.css:124]: I've a habit of using shorthand while giving margin and padding it's a good practice as I've heard from many senior mentors.

**Comment by nehalskdev** [comment]
> Looks good brother
> u can make it better using some other font

You can suggest me a good font which may look better?

**Comment by MTahir001** [comment]
> > Looks good brother
> > u can make it better using some other font
> 
> You can suggest me a good font which may look better?

Sure brother

I think Poppins will looks good

File: index.html (hunk 1 above)
Comment (murtazshaikh) [reply index.html:33]: I don't know about this # but for link to open in new tab we use "_blank" https://www.w3schools.com/tags/att_a_target.asp

File: style.css (hunk 3 above)
Comment (murtazshaikh) [reply style.css:124]: No issue

**Review by murtazshaikh — APPROVED** [review]
LGTM!

**Pull request merged**